apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -1341,6 +1341,88 @@ describe('[Chat]', function () {
 			simpleMessageId = res.body.message._id;
 		});
 
+		it('should fail updating a message if no room id is provided', () => {
+			return request
+				.post(api('chat.update'))
+				.set(credentials)
+				.send({
+					msgId: message._id,
+					text: 'This message was edited via API',
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(400)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', false);
+					expect(res.body).to.have.property('errorType', 'invalid-params');
+				});
+		});
+
+		it('should fail updating a message if no message id is provided', () => {
+			return request
+				.post(api('chat.update'))
+				.set(credentials)
+				.send({
+					roomId: testChannel._id,
+					text: 'This message was edited via API',
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(400)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', false);
+					expect(res.body).to.have.property('errorType', 'invalid-params');
+				});
+		});
+
+		it('should fail updating a message if no room id is provided', () => {
```

Comment:
This is duplicated with the first one added in this file...

Reply:
Oops, this should test the case where text is not provided
Tks for the heads up!